lib/artemis_api/client.rb

```diff
@@ -51,8 +51,12 @@ def find_all(type, facility_id: nil, batch_id: nil, include: nil, filters: nil)
             else
               "#{@options[:base_uri]}/api/v3/#{type}"
             end
+
       url = "#{url}?include=#{include}" if include
-      url = "#{url}?#{format_filters(filters)}" if filters
+      if filters
+        formatted_filters = format_filters(filters)
+        url = (include) ? "#{url}&#{formatted_filters}" : "#{url}?#{formatted_filters}"
```

Comment:
should we do this with `URI` so that you can add more query params (like `fields`) in the future?